### PR TITLE
Add basic feature issue template

### PR DESCRIPTION
**Pull request by tatyree**
This includes some optional sections and boilerplate.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-work-template.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: Feature work template
+about: This is the standard template for creating new technical work for featrues
```
Comment (cpjmcquillan) [inline .github/ISSUE_TEMPLATE/feature-work-template.md:3]: features

File: .github/ISSUE_TEMPLATE/feature-work-template.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+
+(optional - url is an example only) [Link to project planner](https://dfedigital.atlassian.net/browse/TEVA-1010) 
+
+## Background
+
+(required) Why are we doing this work? Describe the thinking, the user need and who was involved.  For people without GitHub accounts please use their position instead of their name (i.e. 'Product Manager' or 'Product').  Make it as detailed as necessary.  
+
+## Design/IA (optional showing boilerplate)
```
Comment (david-mears-dfe) [inline .github/ISSUE_TEMPLATE/feature-work-template.md:16]: Does anyone know what 'optional showing boilerplate' means?

File: .github/ISSUE_TEMPLATE/feature-work-template.md (hunk 1 above)
Comment (david-mears-dfe) [reply .github/ISSUE_TEMPLATE/feature-work-template.md]: fix'd

File: .github/ISSUE_TEMPLATE/feature-work-template.md (hunk 2 above)
Comment (david-mears-dfe) [reply .github/ISSUE_TEMPLATE/feature-work-template.md:16]: Does it mean 'this is optional. here is a boilerplate'?

**Pull request merged**